platform/android/export/export.cpp

```diff
@@ -1341,6 +1356,40 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
+
+	// Creates strings.xml files inside the gradle project for different locales.
+	void _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset) {
+		const char *locales[] = {
+			"ar", "bg", "ca", "cs", "da", "de", "el", "en", "es", "fa", "fi", "fr", "hi", "hr",
+			"hu", "in", "it", "iw", "ja", "ko", "lt", "lv", "nb", "nl", "pl", "pt", "ro", "ru",
+			"sk", "sl", "sr", "sv", "th", "tl", "tr", "uk", "vi", "zh", "zh_HK", "zh_TW", "es-ES"
```

Comment:
Move "es-ES" to the alphabetical location. Also, should it be '_' or '-'

Reply:
it should be _